pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>uk.me.jstott</groupId>
   <artifactId>jcoord</artifactId>
   <packaging>jar</packaging>
-  <version>1.1-xni06-20170214</version>
+  <version>1.3-xni06-20180430</version>
```

Comment:
Shouldn't this be today's date?

Reply:
Good point, apologies for not fixing that. Now fixed.